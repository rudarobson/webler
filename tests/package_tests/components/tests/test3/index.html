<!--components:
	import simple-component;
-->
<!DOCTYPE html>
<html>

<body>
	<simple-component class="substitute-blah" merged-empty-attribute merged-empty-attribute2 merged-nonempty-attribute="a value"></simple-component>
</body>

</html>
