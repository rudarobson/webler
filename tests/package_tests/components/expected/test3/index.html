<!DOCTYPE html><html><body><div class="blah substitute-blah" merged-empty-attribute merged-empty-attribute2 merged-nonempty-attribute="a value"></div></body></html>